example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=C:\flutter\flutter"
-export "FLUTTER_APPLICATION_PATH=C:\Users\Shady\flutter_clean_architecture\example"
-export "FLUTTER_TARGET=lib\main.dart"
+export "FLUTTER_ROOT=/Users/rafaelcmm/SDKs/flutter"
+export "FLUTTER_APPLICATION_PATH=/Users/rafaelcmm/Workspace/flutter_clean_architecture/example"
+export "FLUTTER_TARGET=lib/main.dart"
 export "FLUTTER_BUILD_DIR=build"
-export "SYMROOT=${SOURCE_ROOT}/../build\ios"
+export "SYMROOT=${SOURCE_ROOT}/../build/ios"
 export "OTHER_LDFLAGS=$(inherited) -framework Flutter"
-export "FLUTTER_FRAMEWORK_DIR=C:\flutter\flutter\bin\cache\artifacts\engine\ios"
+export "FLUTTER_FRAMEWORK_DIR=/Users/rafaelcmm/SDKs/flutter/bin/cache/artifacts/engine/ios"
```

Comment:
Is that correct? Should it be your computer's address?

Reply:
This is auto-generated by flutter and doesn't have any impact. It changes when other person runs the example.